### PR TITLE
Add __r*__ operators (e.g. radd).

### LINKED ISSUE #152: Document somewhere that the order of the operations matter
This took almost 2 hours of my afternoon yesterday:

```python
import numpy as np
import biggus
arr = biggus.NumpyArrayAdapter(np.array([1, 2, 3.]))

1 + arr

TypeError: unsupported operand type(s) for +: 'int' and 'NumpyArrayAdapter'
```

But biggus is OK with this:

```python
arr + 1
<_Elementwise shape=(3) dtype=dtype('float64')>

(arr + 1).ndarray()
array([ 2.,  3.,  4.])
```

### PR DESCRIPTION
**Pull request by pelson**
Closes #152.

### CODE REVIEW

**Comment by pelson** [comment]
Ping @ocefpaf.

File: biggus/__init__.py [hunk 1]
```diff
@@ -860,6 +914,7 @@ def __init__(self, array, broadcast, leading_shape=()):
         shape = self._shape_from_broadcast_dict(self.array.shape, broadcast)
         for length in leading_shape:
             if length < 1:
+                print leading_shape
```
Comment (ocefpaf) [inline biggus/__init__.py]: `()` please!

Comment (ocefpaf) [inline biggus/__init__.py]: Actually I think that is a leftover "debugging" print, right?  I'd rather if length was printed in the `ValueError` below.

Comment (pelson) [inline biggus/__init__.py]: Woops :wink:

**Pull request merged**